README.md

```diff
@@ -45,22 +52,16 @@ NUnit is made up of several projects. When reporting issues, please try to repor
 ### Core Projects ###
 
 - [NUnit Test Framework](https://github.com/nunit/nunit) - The test framework used to write NUnit tests (this repository)
-- [NUnit Console and Engine](https://github.com/nunit/nunit-console) - Runs unit tests from the command line and provides the engine that is used by other test runners to run NUnit tests
+- [NUnit Visual Studio Adapter](https://github.com/nunit/nunit3-vs-adapter) - Visual Studio/Dotnet adapter for running NUnit 3 and 4 tests in Dotnet and Visual Studio.
+- [NUnit Console and Engine](https://github.com/nunit/nunit-console) - Runs unit tests from the command line and provides the engine that is used by other test runners to run NUnit tests.
 
 ### Visual Studio Extensions ###
 
-- [NUnit 3 Visual Studio Adapter](https://github.com/nunit/nunit3-vs-adapter) - Visual Studio adapter for running NUnit 3 tests in Visual Studio and in VSTS/TFS builds
-- [NUnit Visual Studio Templates](https://github.com/nunit/nunit-vs-templates) - Project templates and snippets for writing unit tests in Visual Studio
+- [NUnit Visual Studio Adapter](https://github.com/nunit/nunit3-vs-adapter) - Visual Studio adapter for running NUnit 3 and 4 tests in in Dotnet and Visual Studio.
```

Comment:
question: would `Dotnet` here refer to the CLI tool? If so, I'm wondering if it may read better as 

"Visual Studio adapter for running NUnit 3 and 4 tests in Visual Studio or the `dotnet` command line"

Reply:
Yes, that's better